scss/_normalize.scss

```diff
@@ -0,0 +1,350 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Nice work including normalize. Please can you leave a reply to this comment explaining why you would use Normalize in a project, and what other popular alternatives there are to solve the same problem? Just wanna trigger your brain and research a little haha. Have fun !

Reply:
Why I would use normalize in a project is that it makes browsers render all elements more consistently and in line with modern standards. in other words, it makes built-in browser styling consistent across browsers. One popular alternative is CSS Resets which aim to remove all built-in browser styling